op-e2e/actions/proofs/helpers/env.go

```diff
@@ -68,6 +69,8 @@ func NewL2FaultProofEnv[c any](t helpers.Testing, testCfg *TestCfg[c], tp *e2eut
 			dp.DeployConfig.ActivateForkAtGenesis(rollup.Holocene)
 		case Isthmus:
 			dp.DeployConfig.ActivateForkAtGenesis(rollup.Isthmus)
+		default:
+			panic(fmt.Sprintf("unhandled HF:%v", testCfg.Hardfork))
```

Comment:
This should fail the test instead of panicking.  Otherwise it can wind up making it harder to see what went wrong because whole ton of tests are aborted due to the panic.